components/home/Main.tsx

```diff
@@ -0,0 +1,74 @@
+import React from "react";
+import styled from "@emotion/styled";
+import CategoryCarousel from "./CategoryCarousel";
+import { colors } from "../../lib/constants/colors";
+
+const Styled = {
+  Root: styled.div`
+    height: calc(100vh - 120px);
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+  `,
+
+  Contents: styled.div`
+    display: flex;
+    align-items: center;
+    margin: 32px 0 72px 0;
+
+    @media (max-width: 768px) {
+      flex-direction: column;
+    }
+  `,
+
+  Headline: styled.div`
+    margin-right: 92px;
+    & > h1 {
+      font-size: 52px;
+      line-height: 66px;
+      font-weight: bold;
+      color: ${colors.green50};
+      & > div {
+        text-decoration-line: line-through;
+        text-decoration-color: ${colors.orange50};
+      }
+      & > strong {
+        color: ${colors.orange50};
+      }
+    }
+    & > h2 {
+      margin-top: 18px;
+      font-weight: bold;
+      font-size: 18px;
+      color: ${colors.green50};
+    }
+  `,
+
+  CategoryCarouselWrapper: styled.div`
+    width: 1200px;
+    padding: 0 16px;
+  `,
+};
+
+function Main() {
+  return (
+    <Styled.Root>
+      <Styled.Contents>
+        <Styled.Headline>
+          <h1>
+            <div>더 이상</div>
```

Comment:
```suggestion
            더 이상<br/>
```
처럼 태그를 없애는 게 더 시맨틱할 것 같아요! 특히 `h1` 내부에 들어가는 내용들은 구글이 민감하게 긁어가는 녀석들이라서 더 신경써주면 좋습니다!

Reply:
오오 사실 `<div>더 이상</div>` 으로 감싼건 디자인처럼 "더 이상"에만 가운데 짝대기을 적용하기 위해선데요!
혹시 아시는 좋은 방법 있으신가유👀  
뭔가 최대한 다른 의미없이 스타일을 위한 태그만 딱 적용하고 싶네요...!

Reply:
그렇다면 `del` 태그를 써보는 건 어떨까유??? 근데 요거 색상 지정이 되려나 잘 모르겟네용

Reply:
오호 잘 되네요!!
https://github.com/Lubycon/eat-it-all-frontend/pull/7/commits/b2c6de72652a28c751263cf0161802f8bdd5e63d 적용했슴니당🙇‍♂️